### PR TITLE
Style feature

### PR DESCRIPTION
**Pull request by themonster2015**
We added CSS styling to the app to match the general layout.
We used Bootstrap and custom styling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ This is a simple javascript booklist application. The project tests the use of o
 
 [Live Demo Link](https://themonster2015.github.io/awesome-books/)
```
Comment (mimipeshy) [inline README.md:14]: - [x] the live demo link gives a 404 error, kindly fix this

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Team 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- styling follows the wireframe given ✔️ 
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: README.md (hunk 1 above)
Comment (themonster2015) [reply README.md:14]: thanks for noticing it. We've fixed it!

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**